enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk_setup/analytics.ts

```diff
@@ -5,66 +5,74 @@ import type {
 } from "metabase-enterprise/embedding_iframe_sdk/types/embed";
 
 import type { SdkIframeEmbedSetupExperience } from "./types";
-import { filterEmptySettings } from "./utils/filter-empty-settings";
 
-export const trackEmbedWizardExperienceSelected = (
+/**
+ * Tracking every embed options would be too much, so we only track
+ * the most relevant options that reveal usage patterns.
+ */
+const EMBED_OPTIONS_TO_TRACK = [
```

Comment:
We now report the embed options after clicking "Get Code" to get to the get code step, so we should not bloat the event_detail too much. I think these is the essential to understand how people is using embeds.